Update double-key-encryption.md (#3798)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/double-key-encryption.md

Ensure that no errors appear in the output. When you're ready, [register your key store](#register-your-key-store).

> > [!NOTE]

Reviewer: ```suggestion
```

> > Key name is case senstive. Provide the Keyname as in the appsettings.json.

Reviewer: ```suggestion
> The key name is case sensitive. Enter the key name as it appears in the appsettings.json file.
```

> > The key name is case sensitive. Enter the key name as it appears in the appsettings.json file.

Reviewer: ```suggestion
The key name is case sensitive. Enter the key name as it appears in the appsettings.json file.
```

Reviewer: As per @KCCross's suggestion.
